mautic/mautic
# Segment email send to email address just once

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:

Our client want use contacts with same email address, but avoid send more times to that email address, even If contact's are in segments. For example If more employees use same email address, or family use one email address for parents and children.
This PR added new configuration parameter to enable avoid send segment email to more like one unique email address.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create segment
3. Create 3 contacts with same  email@domain.tld and add them to segment
4. Create segment email
4. Go to configuration and enable new options - Send to duplicated just once 
![image](https://user-images.githubusercontent.com/462477/126951145-106dd133-282d-4871-8954-c9d3123db006.png)
5. Send segment email
6. See just one email you receive

5. 


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9781"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
